### DISCUSSION TITLE
Media Impression Data

### DISCUSSION THREAD

**Discussion by DongHarry-Kang** [google/lightweight_mmm#270] [Q&A]
In Robyn, if we do not have media impression data, such as TV ads, we can use cost data instead; for Lightweight, if we do not have good impression data, could we use cost data to do simulation?

**Comment by statm3n** [comment]
You are right. Impression or cost data are generally the two data sources used as media exposure variables in MMM.

**Reply by DongHarry-Kang** [reply]
Thank you very much. Could you confirm whether we can use spend data as impression data in Lightweight MMM?
